stock_quant_partner_info/README.rst

```diff
@@ -0,0 +1,43 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========================
+Stock - Quant partner info
+==========================
+
+With this module you can see in which is the related partner if it is in a supplier or
```

Comment:
With this module, when you are in a customer or supplier location, the name of the partner of the last picking to move the quant is displayed.